test/unit/v2.0.0/converter-v2-to-v1.test.js

```diff
@@ -505,4 +505,78 @@ describe('v2.0.0 to v1.0.0', function () {
             });
         });
     });
+
+    describe('retainIds', function () {
+        it('should handle IDs correctly when set to true', function () {
+            transformer.convert({
```

Comment:
Tests will need to be added for `retainIds` in `itemGroup`, `item` and `response`